--- PKGBUILD	2023-12-09 09:54:18.710600613 +0100
+++ PKGBUILD	2023-12-09 10:52:55.345430496 +0100
@@ -9,3 +9,9 @@
 arch=('x86_64' 'aarch64')
 url='https://electronjs.org/'
 depends=("${_pkgname}${_major}-bin")
+
+package() {
+	mkdir -p "$pkgdir/usr/bin" "$pkgdir/usr/lib"
+	ln -sf "${_pkgname}${_major}" "$pkgdir/usr/bin/$pkgname"
+	ln -sf "${_pkgname}${_major}" "$pkgdir/usr/lib/$pkgname"
+}
\ No newline at end of file
