diff --git a/PKGBUILD b/PKGBUILD
index 1a4f782..05c3172 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -29,13 +29,14 @@ arch=(
   'armv7h')
 license=(LGPL)
 depends=(
-  "gtk4>=4.11")
+  "gtk4-git>=4.11")
 makedepends=(
   cmake
   git
   meson
   gi-docgen
   sassc
+  glib2-devel
   gobject-introspection
   vala)
 checkdepends=(
@@ -46,7 +47,7 @@ source=(
 sha256sums=(SKIP)
 
 pkgver() {
-  cd "${_pkgname}"
+  cd "$srcdir/${_pkgname}"
   git describe --tags | sed 's/-/+/g'
 }
 
@@ -56,6 +57,7 @@ build() {
 }
 
 check() {
+  bash -c ' \
   export XDG_RUNTIME_DIR="$PWD/runtime-dir" \
          WAYLAND_DISPLAY=wayland-5
 
@@ -65,12 +67,13 @@ check() {
          --idle-time=0 &
   _w=$!
 
-  bash trap "kill ${_w}; wait" EXIT
+  trap "kill ${_w}; wait" EXIT
 
   meson \
     test \
       -C build \
       --print-errorlogs
+  '
 }
 
 _pick() {
