--- PKGBUILD.orig	2024-03-26 20:33:26.262424172 +0100
+++ PKGBUILD	2024-03-26 20:36:53.513605903 +0100
@@ -19,9 +19,23 @@
     echo r$(git rev-list --count main).$(git rev-parse --short main)
 }
 
+build() {
+	cd "$srcdir"/just-perfection/
+    bash scripts/build.sh
+}
+
 package() {
     cd "$srcdir"/just-perfection/
-    bash scripts/build.sh
-    mkdir -p "$pkgdir/usr/share/gnome-shell/extensions/just-perfection-desktop@just-perfection"
-    unzip just-perfection-desktop@just-perfection.shell-extension.zip -d "$pkgdir/usr/share/gnome-shell/extensions/just-perfection-desktop@just-perfection"
+
+	local uuid=$(grep -Po '(?<="uuid": ")[^"]*' src/metadata.json)
+
+	install -d "$pkgdir/usr/share/gnome-shell/extensions/${uuid}"
+	bsdtar -xvf "${uuid}.shell-extension.zip" -C \
+	  "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/" --no-same-owner
+
+	mv "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/locale" "$pkgdir/usr/share/"
+
+	install -Dm644 src/schemas/org.gnome.shell.extensions.just-perfection.gschema.xml -t \
+	  "$pkgdir/usr/share/glib-2.0/schemas/"
+	rm -rf "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/schemas/"
 }
