--- PKGBUILD.orig	2024-03-26 17:26:14.526321427 +0100
+++ PKGBUILD	2024-03-26 17:27:58.671192063 +0100
@@ -7,7 +7,7 @@
 # Contributor: Jakub Schmidtke <sjakub@gmail.com>
 
 pkgname=waterfox
-pkgver=G6.0.11
+pkgver=$(curl -s 'https://api.github.com/repos/WaterfoxCo/Waterfox/releases/latest' | jq -r '.name')
 pkgrel=2
 pkgdesc='Fork of Mozilla Firefox featuring some privacy, usability, and speed enhancements.'
 arch=(x86_64)
@@ -22,6 +22,7 @@
 	ttf-font
 )
 makedepends=(
+	jq
 	cbindgen
 	clang
 	diffutils
@@ -61,7 +62,7 @@
 	"$pkgname.desktop"
 )
 b2sums=(
-	'861e981cd9a7a0a497e7f3a752a435d63825ae4227abfffbf8019387432b44eaf950360ff06968cf65f74d3f7396a3e168e9c8c97ba3be1f1bea14011746ba58'
+	'SKIP'
 	'b40c9d21c97f1b1ac9cae61593427c18f2262e39484874ad2077be7484f929ad7785406541f1b964041f6cd43f1fdd5c98e4942af3586de67d7b9c50a829a996'
 )
 
