diff --git a/PKGBUILD b/PKGBUILD
index c7d6bb5..46858ad 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -5,7 +5,9 @@ pkgver=v0.2.20.r0.4c7fd88d57cd
 pkgrel=1
 pkgdesc="A third-party, open-source ProtonMail CardDAV, IMAP and SMTP bridge"
 arch=('x86_64')
+depends=('glibc')
 makedepends=('git' 'go')
+provides=('hydroxide')
 conflicts=('hydroxide')
 url="https://github.com/emersion/hydroxide"
 license=('MIT')
@@ -18,6 +20,13 @@ pkgver() {
 }
 
 build() {
+	# https://wiki.archlinux.org/title/Go_package_guidelines
+	export CGO_CPPFLAGS="${CPPFLAGS}"
+	export CGO_CFLAGS="${CFLAGS}"
+	export CGO_CXXFLAGS="${CXXFLAGS}"
+	export CGO_LDFLAGS="${LDFLAGS}"
+	export GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=readonly -modcacherw"
+
 	cd "$srcdir/hydroxide"
 	go build "./cmd/hydroxide"
 }
