--- PKGBUILD.orig	2024-03-18 17:48:01.019805327 +0100
+++ PKGBUILD	2024-03-18 17:06:20.615884098 +0100
@@ -10,11 +10,11 @@
 depends=('libadwaita' 'libgee' 'libpqmarble' 'vte4')
 makedepends=('meson' 'vala')
 checkdepends=('appstream-glib')
-source=("$pkgname-$pkgver.tar.gz::$url/-/archive/v$pkgver/blackbox-v$pkgver.tar.gz")
-sha256sums=('0a8b76666ac71a6dc9e05b8084e1c6c53cda880ad1f8f778d9d5de9a6644ad0d')
+source=("$pkgname::git+$url.git")
+sha256sums=('SKIP')
 
 build() {
-  arch-meson "blackbox-v$pkgver" build \
+  arch-meson "blackbox-terminal" build \
     -Dblackbox_is_flatpak=false
   meson compile -C build
 }
