--- PKGBUILD.orig	2024-03-26 20:14:42.846199253 +0100
+++ PKGBUILD	2024-03-26 20:30:44.415282264 +0100
@@ -1,9 +1,9 @@
 # Maintainer: Nep_Nep <nepnep91 at child dot pizza>
 # Contributor: Tim Paik <timpaik@163.com>
 _extname=quick-settings-tweaks
-_uuid=$_extname@qwreey
+#_uuid=$_extname@qwreey
 pkgname=gnome-shell-extension-$_extname-git
-pkgver=r235.a9bd1ae
+pkgver=r243.79e3013
 pkgrel=1
 pkgdesc="A Gnome 43+ extension which allows you to customize the new Quick Settings Panel to your liking!"
 arch=('any')
@@ -28,7 +28,15 @@
 
 package() {
   cd "$srcdir/$_extname"
-  install -d "$pkgdir/usr/share/gnome-shell/extensions/$_uuid"
-  bsdtar xvf "dist/$_uuid.shell-extension.zip" -C "$pkgdir/usr/share/gnome-shell/extensions/$_uuid"
-  chown root:root -R "$pkgdir/usr/share/gnome-shell/extensions/$_uuid"
+  local uuid=$(grep -Po '(?<="uuid": ")[^"]*' src/metadata.json)
+
+  install -d "$pkgdir/usr/share/gnome-shell/extensions/${uuid}"
+  bsdtar -xvf "dist/${uuid}.shell-extension.zip" -C \
+    "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/" --no-same-owner
+
+  mv "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/locale" "$pkgdir/usr/share/"
+
+  install -Dm644 src/schemas/org.gnome.shell.extensions.quick-settings-tweaks.gschema.xml -t \
+    "$pkgdir/usr/share/glib-2.0/schemas/"
+  rm -rf "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/schemas/"
 }
