--- vencord-desktop-git/PKGBUILD	2023-08-31 13:54:17.301065364 +0200
+++ ../PKGBUILD	2023-08-31 13:53:27.970473808 +0200
@@ -18,10 +18,12 @@
 provides=("vencord")
 conflicts=("vencord")
 
-source=("$pkgname::git+$url.git" "vencord-desktop.desktop")
+source=("$pkgname::git+$url.git" "vencord-desktop.desktop" "vencorddesktop.sh" "package.json.patch")
 
 sha256sums=('SKIP'
-            'eb606c9532a79519f6e72b5a1bec2776c47e13321e3fd986ab57edaa9cb7773a')
+            'eb606c9532a79519f6e72b5a1bec2776c47e13321e3fd986ab57edaa9cb7773a'
+            'c0ced11fca2c1aed8691a45c50cb5550b75e21cb9cb3e38d1b7135ab91d8eca1'
+            '2fbdc76bfe550242ed7dd16fa6aafabc49f6b460b034dbbb5b4836809b0b0f05')
 
 pkgver() {
   cd "$pkgname"
@@ -30,6 +32,7 @@
 
 build() {
   cd "$pkgname"
+  patch -u < "$srcdir"/package.json.patch
 
   pnpm i
   pnpm package:dir
@@ -37,14 +40,12 @@
 
 package() {
   cd "$srcdir"
-
-  install -d "$pkgdir"/opt/vencord
-  cp -R "$pkgname/dist/linux-unpacked/." "$pkgdir/opt/vencord"
+  install -d "$pkgdir"/usr/{lib/vencord,bin}
+  cp "$pkgname/dist/linux-unpacked/resources/app.asar" "$pkgdir/usr/lib/vencord"
 
   install -Dm 644 "vencord-desktop.desktop" "$pkgdir/usr/share/applications/vencord-desktop.desktop"
   install -Dm 644 "$pkgname/static/icon.png" "$pkgdir/usr/share/pixmaps/vencord.png"
   install -Dm 644 "$pkgname/LICENSE" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
-
-  install -d "$pkgdir"/usr/bin
-  ln -s /opt/vencord/vencorddesktop "$pkgdir"/usr/bin/vencord-desktop
+  
+  install -Dm 755 vencorddesktop.sh "${pkgdir}"/usr/bin/vencord-desktop
 }
