diff --git a/PKGBUILD b/PKGBUILD
index 629e550..d8d07f8 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,7 @@ pkgdesc="A GNOME web browser based on the WebKit rendering engine."
 url="https://wiki.gnome.org/Apps/Web"
 arch=('x86_64')
 license=('GPL')
-depends=(webkitgtk-6.0 gcr-4 gtk4 libdazzle libhandy libarchive libportal-gtk4 libadwaita)
+depends=(webkitgtk-6.0 gcr-4 gtk4-git libdazzle libhandy libarchive libportal-gtk4 libadwaita-git)
 makedepends=(cmake docbook-xml startup-notification lsb-release gobject-introspection
              yelp-tools git meson)
 checkdepends=(xorg-server-xvfb aspell hspell hunspell nuspell libvoikko)
