--- PKGBUILD.orig	2024-03-26 18:39:09.435993008 +0100
+++ PKGBUILD	2024-03-26 18:44:53.037706246 +0100
@@ -2,7 +2,7 @@
 
 pkgname=gnome-shell-extension-panel-corners-git
 _pkgname=panel-corners
-pkgver=6.r0.gc6a046d
+pkgver=10.r2.g9ae1377
 pkgrel=1
 pkgdesc="A gnome-shell extension to keep the old topbar corners, which were removed for GNOME 42"
 arch=('any')
@@ -23,13 +23,20 @@
 build() {
   cd "$_pkgname"
 
-  make pkg
+  make build
 }
 
 package() {
   cd "$_pkgname"
 
   local uuid=$(grep -Po '(?<="uuid": ")[^"]*' metadata.json)
+
   install -d "$pkgdir/usr/share/gnome-shell/extensions/${uuid}"
-  bsdtar -xvf pkg/${uuid}.zip -C "$pkgdir/usr/share/gnome-shell/extensions/${uuid}"
+  bsdtar -xvf "build/${uuid}.shell-extension.zip" -C \
+    "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/" --no-same-owner
+
+  install -Dm644 schemas/org.gnome.shell.extensions.panel-corners.gschema.xml -t \
+    "$pkgdir/usr/share/glib-2.0/schemas/"
+  rm -rf "$pkgdir/usr/share/gnome-shell/extensions/${uuid}/schemas/"
+
 }
